packages/shared-ux/router/impl/router.test.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Route } from './router';
+
+describe('Route', () => {
+  test('renders', () => {
```

Comment:
Can we add a more extensive test suite? Given that the render logic depends on a lot of props, would be nice to test some of those scenarios.

Reply:
I've added some of the react dom route props in the test suite - let me know what you think. I think for testing `exact`, `sensitive` and `strict` it might involve an integration test?